addons/website_forum/views/forum_forum_templates_post.xml

```diff
@@ -103,475 +126,456 @@
+                <div class="btn-toolbar align-items-center mt-3" role="toolbar">
+                    <t t-if="post_id.state == 'active' and post_id.active != False" t-call="website_forum.vote">
+                        <t t-set="post" t-value="post_id"/>
+                        <t t-if="post_id == answer">
+                            <t t-set="helper_accept">Mark as Best Answer</t>
+                            <t t-set="helper_own_post">You can't vote for your own post</t>
+                            <t t-set="helper_no_karma">You don't have enough karma</t>
+                            <t t-set="helper_decline">Unmark as Best Answer</t>
+                            <a t-if="question.can_answer and question.forum_id.mode == 'questions'"
+                                t-attf-class="o_wforum_validate_toggler btn #{not answer.is_correct and answer.create_uid.id != uid and post_id.can_accept and 'opacity-50 opacity-100-hover'} #{not post_id.can_accept and 'karma_required opacity-25 '}#{answer.create_uid.id == uid and 'opacity-25'}"
+                                href="#"
+                                t-attf-data-karma="#{post_id.karma_accept}"
+                                t-att-data-helper-accept="helper_accept"
+                                t-att-data-helper-own-post="helper_own_post"
+                                t-att-data-helper-no-karma="helper_no_karma"
+                                t-att-data-helper-decline="helper_decline"
+                                t-att-title="not post_id.can_accept and helper_no_karma or answer.is_correct and helper_decline or not answer.is_correct and helper_accept or answer.is_correct and answer.create_uid.id == uid and helper_own_post"
                                 data-bs-toggle="tooltip"
+                                t-attf-data-target="#answer-#{post_id.id}"
+                                t-attf-data-href="/forum/#{ slug(question.forum_id) }/post/#{ slug(answer) }/toggle_correct">
+                                <i t-attf-class="fa #{answer.is_correct and 'fa-check-circle text-success' or 'fa-check-circle-o'} fs-5"/>
+                            </a>
+                        </t>
+                    </t>
+                    <div class="d-flex align-items-center ms-auto">
+                        <t t-if="post_id == question">
+                            <t t-set="user_answer" t-value="uid and post_id.child_ids.filtered(lambda a: a.create_uid.id == uid)"/>
+                            <div t-if="not user_answer and uid and question.can_answer and question.state == 'active' and question.active != False" class="d-flex ms-auto me-2">
+                                <a t-attf-class="btn btn-link collapsed #{ not question.can_answer and 'karma_required text-muted' }#{ forum.has_pending_post and 'disabled' } text-decoration-none"
+                                    t-att-data-karma="question.forum_id.karma_answer"
```

Comment:
Styling note: I suppose you clse the "a" on a given line to split the parameters from the i. According to me ( :D à you can align t-att vertically (t-attf-class, then t-att-data-karma, data-bs-toggle, ...), my linter agrees with it. Then you can close the tag in the same line as the href, and have the i being "not aligned" with the above parameters.